packages/router/src/grpc/custody/authorize.ts

```diff
@@ -32,14 +33,27 @@ export const authorize: Impl['authorize'] = async (req, ctx) => {
   if (!passwordKey) throw new ConnectError('User must login to extension', Code.Unavailable);
 
   const wallets = await local.get('wallets');
-  const { encryptedSeedPhrase } = wallets[0]!.custody;
+  const {
+    custody: { encryptedSeedPhrase },
+    fullViewingKey,
+  } = wallets[0]!;
+
+  if (!fullViewingKey)
+    throw new ConnectError('Unable to get full viewing key', Code.Unauthenticated);
```

Comment:
Let me know if `Code.Unauthenticated` is the correct code to be using here. It seemed appropriate to me.